root/etc/e-smith/events/actions/nethserver-tancredi-conf

```diff
@@ -33,6 +33,22 @@ if [[ ! -f ${dst_file} ]]; then
     echo 'hostname = "'$(hostname)'"' >> ${dst_file}
     echo 'provisioning_url_scheme = "https"' >> ${dst_file}
 
+    # set ldap defaults
+    echo 'ldap_server = "'$(hostname)'"' >> ${dst_file}
+    echo 'ldap_port = "10389"' >> ${dst_file}
+    echo 'ldap_tls = "none"' >> ${dst_file}
+    echo 'ldap_user = ""' >> ${dst_file}
+    echo 'ldap_password = ""' >> ${dst_file}
+    echo 'ldap_base = ""' >> ${dst_file}
+    echo 'ldap_name_display = "%cn %o"' >> ${dst_file}
```

Comment:
Ensure this format suits all phone vendors!

The value format of this variable is still not clearly defined. 

- Is it a string supporting placehoders (like %o)? 
- Is it just a list of LDAP attributes that are joined together?

Reply:
right now, 
it is "%cn %o" for Yealink, Snom
is id "cn" for Sangoma, Gigaset, Fanvil